realm/realm-library/src/main/java/io/realm/RealmObject.java

```diff
@@ -354,6 +365,30 @@ public final boolean load() {
     }
 
     /**
+     * Adds a change listener to a RealmObject.
```

Comment:
That is how the current change listeners work? The fine-grained object notifications work slightly different that way